server/database/models/admin.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+const adminSchema = new Schema({
+  id: {
```

Comment:
and in here, we do not want the id in here because MongoDB create an id for all instance as this name "_id"